Button/Button.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * MyTheme styled button components and behaviors.
+ *
+ * @example
+ * <Button>Hello Enact!</Button>
+ *
+ * @module my-theme/Button
+ * @exports Button
+ * @exports ButtonBase
+ * @exports ButtonDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Spottable from '@enact/spotlight/Spottable';
+import {ButtonBase as UiButtonBase, ButtonDecorator as UiButtonDecorator} from '@enact/ui/Button';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+
+import Icon from '../Icon';
+import Skinnable from '../Skinnable';
+
+import componentCss from './Button.module.less';
+
+/**
+ * A button component.
+ *
+ * This component is most often not used directly but may be composed within another component as it
+ * is within [Button]{@link my-theme/Button.Button}.
+ *
+ * @class ButtonBase
+ * @memberof my-theme/Button
+ * @extends ui/Button.ButtonBase
+ * @ui
+ * @public
+ */
+const ButtonBase = kind({
+	name: 'Button',
+
+	propTypes: /** @lends my-theme/Button.ButtonBase.prototype */ {
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal Elements and states of this component.
+		 *
+		 * The following classes are supported:
+		 *
+		 * * `button` - The root class name
+		 * * `bg` - The background node of the button
+		 * * `selected` - Applied to a `selected` button
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		css: PropTypes.object,
+
+		/**
+		 * Provides a way to call special interface attention to this button. It will be "featured"
+		 * in some way by the theme's visual rules.
+		 *
+		 * @type {Boolean}
+		 * @public
+		 */
+		selected: PropTypes.bool
```

Comment:
I forgot to note on this line that we discussed adding some inline comments on why this prop is here.